demo/style.css

```diff
@@ -29,93 +29,100 @@ pre {
 }
 
 code {
+  display: block;
   font-family: Consolas, Monaco, monospace;
   font-size: 10pt;
+  overflow-x: auto;
+  padding: 0.5em;
```

Comment:
Isn't this already coming from the themes?

Reply:
Ah, yes. I added it here while I was highlighting asynchronously with the worker so the code blocks don't jump around.

Reply:
But the CSS should have nothing to do with the worker...?